blockmanager.go

```diff
@@ -2736,6 +2736,11 @@ func (b *blockManager) checkHeaderSanity(blockHeader *wire.BlockHeader,
 func (b *blockManager) calcNextRequiredDifficulty(newBlockTime time.Time,
 	reorgAttempt bool) (uint32, error) {
 
+	// Some networks don't support difficulty retargeting.
+	if b.cfg.ChainParams.PoWNoRetargeting {
```

Comment:
I think we want to go with this PR instead: https://github.com/lightninglabs/neutrino/pull/263

Then we just share all this code, and don't need to make changes in two areas.